lib/ifupdown.sh.in

```diff
@@ -29,10 +40,10 @@ esac
 
 # Previous work (create the interface)
 if [ "$MODE" = 'start' ] && [ ! -d "/sys/class/net/$IFACE" ]; then
-  brctl addbr "$IFACE" || exit 1
+  "$brctl" addbr "$IFACE" || exit 2
```

Comment:
Another good catch!  I actually intended to use `exit 2` in commit 5846816 ... but somewhere along the way I lost that.  I will fix this too.

I typically use `exit 1` for generic "soft" errors that can be corrected by running the program differently: bad arguments, bad environment, etc.  I typically use `exit 2` for generic "hard" errors that cannot be corrected simply by running the program differently: missing installation files, missing kernel support, etc.  This distinction might be useful if someone is actually checking the exit codes from the script.